packages/discord-embedder/tsconfig.json

```diff
@@ -7,15 +7,32 @@
     "moduleResolution": "Bundler",
```

Comment:
This entire file needs to be changed to extend the file at /config/tsconfig.base.json